### ISSUE TITLE
Date Range Selection Cell Manipulation

### ISSUE THREAD

**Issue by mrjimoy** [patchthecode/JTAppleCalendar#97]
Hi, finally I found a way to make it as a date range, but there still some gap between dates (white spaces) :

![image](https://cloud.githubusercontent.com/assets/6197219/17316274/beffdc64-589c-11e6-9e71-1bc6cfd3bf61.png)

I would like to make it like : 

![image](https://cloud.githubusercontent.com/assets/6197219/17316279/cd1fc0c0-589c-11e6-922b-051cf4de5353.png)

This is the code to accomplish that (your code with some modification) : 

![image](https://cloud.githubusercontent.com/assets/6197219/17316321/23d7f824-589d-11e6-9b04-52716ac8388c.png)

So basically I add 2 UIView on the CellView, a half on the left side (leftSelectedView) and a half on the right side (rightSelectedView).

How to close that gap? Thanks.

**Comment by patchthecode** [comment]
Whats your cell inset?
`calendarView.cellInset = ?`

**Comment by mrjimoy** [comment]
@patchthecode Thank you, it's worked now :)

**Comment by patchthecode** [comment]
Hey, your calendar looks sorta nice. If you dont mind, do you wanna place an image of it at this link ? -> `https://github.com/patchthecode/JTAppleCalendar/issues/2` 

Pretty images helps others know the sort of cool stuff they can do with this library.
If you don't then its cool either way.

**Comment by techmehra** [comment]
can you share your code please

**Comment by nahub96** [comment]
Can you share it please?

**Comment by sathish-ajira** [comment]
i am also need this feature.can someone share this code please.

**Comment by Dzoni89** [comment]
Hi, having the same issue as mrjimoy but there doesn't seem to be a property named cellInsets on calendarView. Not sure if you've renamed it in an update or?
Also thanks for the best iOS calendar out there, you've saved me a lot of time. + star'd 👍

**Comment by Dzoni89** [comment]
Hi, having the same issue as mrjimoy but there doesn't seem to be a property named cellInsets on calendarView. Not sure if you've renamed it in an update or?
Also thanks for the best iOS calendar out there, you've saved me a lot of time. + star'd 👍

**Comment by patchthecode** [comment]
@Dzoni89 yes, every thing has been renamed to exactly match Apple's UICollectionView's function names and variables.  This makes it easier for developers to find the names they need without trying to remember the weird names i invent.

```
calendarView.minimumLineSpacing = ?
calendarView.minimumInteritemSpacing = ?
```

**Comment by Dzoni89** [comment]
![screen shot 2017-09-28 at 7 42 33 am](https://user-images.githubusercontent.com/20557067/30950999-d4d53dce-a420-11e7-9c9d-bf3ea8d329bf.png)

@patchthecode  thanks for the reply, I've actually tried setting minimumLineSpacing and minimumInterimSpacing to 0 prior to posting here. But now that I've given each cell a border I see that there is no cell spacing, my top and bottom border whose widths are set to equal cell.frame.size.width seem to have some weird spacing. Screwed up with the math somewhere probably, cheers. :/

**Comment by patchthecode** [comment]
@Dzoni89 if you are free, you can contact me here -> https://gitter.im/patchthecode/JTAppleCalendar
I am not understanding the problem correctly.

**Comment by JolChrSA** [comment]
> can you share your code please
